addons/web_editor/static/src/js/wysiwyg/widgets/image_crop.js

```diff
@@ -97,6 +97,59 @@ export class ImageCrop extends Component {
         }
     }
 
+    /**
+     * Crops image with the aspect ratio 1:1
+     */
+    async autoRatio(ratio, cropped) {
```

Comment:
Need to rename the function to align it's working, something like `cropSquare` or something else...
As well as define params in comment.

Reply:
Still, we can think of other names..
 `autoRatio` seems to be confusing to understand the purpose of the function.